src/taco-utils/tacoPackageLoader.ts

```diff
@@ -88,6 +89,7 @@ module TacoUtility {
 
     export interface ITacoPackageLoader {
         lazyRequire<T>(packageName: string, packageId: string, logLevel?: InstallLogLevel): Q.Promise<T>;
+        lazyRun(packageName: string, packageId: string, commandName: string): Q.Promise<string>;
```

Comment:
Would it be worthwhile to take an additional parameter for logLevel - instead of always defaulting to InstallLogLevel.warn?